src/main/java/com/minecolonies/coremod/network/messages/RemoveBlockMessage.java

```diff
@@ -0,0 +1,97 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.api.util.BlockUtils;
+import com.minecolonies.coremod.client.gui.WindowScan;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.client.gui.WindowScan'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)